packages/rol/example/tempus/example_parabolic_thyravec.hpp

```diff
@@ -881,8 +881,8 @@ class Objective_ParabolicControl : public ROL::DynamicObjective<Real> {
     const Thyra::ConstDetachedVectorView<Real> x(x_vb);
     const Thyra::ConstDetachedVectorView<Real> y(y_vb);
     Real ip(0);
-    Real c = ((x.subDim()==nx_) ? four : two);
-    for (uint i=0; i<x.subDim(); i++) {
+    Real c = ((x.subDim()==(long long)nx_) ? four : two);
```

Comment:
C-style cast.

Reply:
@bartlettroscoe , sure, static casts are probably better, however the issue is that we don't have any explicit `long long` types in ROL.  So static casts with the correct types are the best solution.

Reply:
That's not a static cast; it's a C-style cast.  They [mean different things](https://en.cppreference.com/w/cpp/language/explicit_cast).